buildSrc/src/main/java/Dependencies.kt

```diff
@@ -42,7 +42,7 @@ object Versions {
     // that we depend on directly for the fenix-megazord (and for it's
     // forUnitTest variant), and it's important that it be kept in
     // sync with the version used by android-components above.
-    const val mozilla_appservices = "0.39.1"
+    const val mozilla_appservices = "0.40.0"
```

Comment:
I  think we need to update our version on ac, because we are on "0.39.1"
https://github.com/mozilla-mobile/android-components/blob/4bd8ede8d49b0618ce2bb3629d6b27538bd49962/buildSrc/src/main/java/Dependencies.kt#L30